package/etc/conf.d/filters/cisco/esa.conf

```diff
@@ -0,0 +1,40 @@
+filter f_cisco_esa {
+    match("cisco_esa_12_5", value("fields.sc4s_vendor_product") type(glob))
```

Comment:
this should probably not be version-specific

Reply:
Here we want to differentiate the events based on the host and so we have assigned the **sc4s_vendor_product** as **cisco_esa_12_5**. Hence all the logs coming from that host will be assigned to this vendor_product. Do you suggest we change that? and If yes, What is the condition that we put?

Reply:
just "cisco_esa" should be adequate unless we have a strong reason we don't want to force the customer to track the version of the source they are working with because they are really bad at it

Reply:
Updated